px-app-nav.es6.js

```diff
@@ -240,6 +240,18 @@
         observer: '_handleVerticalExpandedViewChanged'
       },
 
+      /**
+       * The parent container width at which the vertical navigation should be expanded. 
+       * Use a number (e.g. `600`) which will be converted to a pixel value (e.g. '600px').
+       * 
+       * This property will overwrite the `verticalExpanded` property. Avoid data
```

Comment:
Maybe we should combine `vertical-expanded` and `vertical-expanded-at` into a single prop `vertical-expanded-threshold`.

If it's `0`, it behaves as `vertical-expanded` does now. If > 0, then it behaves like `vertical-expanded-at`.

Would require a major version bump to avoid breaking anyone who uses those props.

Reply:
Hmm yeah thats a good idea.